### ISSUE TITLE
Wasmer installed, but doesn't seem to be working

### ISSUE THREAD

**Issue by ritzdevp** [wasmerio/wasmer#2859]
I am using a VirtualBox Ubuntu 20.04.3 LTS virtual machine and tried installing wasmer using the curl command given in the README.

The output on running ``` echo "`wasmer -V` | `rustc -V` | `uname -m`"``` is as follows
```wasmer: command not found | rustc 1.57.0|x86_64```

On running the curl command, I get

```downloading: wasmer-linux-amd64
Latest release: 2.2.1
Downloading archive from https://github.com/wasmerio/wasmer/releases/download/2.2.1/wasmer-linux-amd64.tar.gz
######################################################################## 100.0%##O######################################################################## 100.0%
installing: /home/ritzdevp/.wasmer

gzip: stdin: unexpected end of file
tar: Child returned status 1
tar: Error is not recoverable: exiting now
Updating bash profile /home/ritzdevp/.bashrc
warning: the profile already has Wasmer and has not been changed
sh: 169: /home/ritzdevp/.wasmer/bin/wasmer: not found
error: wasmer was installed, but doesn't seem to be working :(
check:  installed successfully ✓
downloading: wapm-cli-linux-amd64
Latest release: 0.5.4
Downloading archive from https://github.com/wasmerio/wapm-cli/releases/download/v0.5.4/wapm-cli-linux-amd64.tar.gz
######################################################################## 100.0%
installing: /home/ritzdevp/.wasmer

gzip: stdin: unexpected end of file
tar: Child returned status 1
tar: Error is not recoverable: exiting now
```

**Comment by heyjdp** [comment]
Please can you provide information to reproduce this issue

**Comment by heyjdp** [comment]
I have tried to reproduce this today. 

1) I installed Ubuntu 20.04 LTS on VirtualBox. 
2) Then I installed rust using `curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh`
3) Then I installed wasmer using `curl https://get.wasmer.io -sSfL | sh`

The installation worked fine, maybe there was a networking error when you tried to retrieve the tar ball.

I did however hit error: `error while loading shared libraries libxkbcommon.so.0` which is well documented in #2822 and #2716 

Closing, cannot reproduce
